src/components/pages/Volunteers/index.tsx

```diff
@@ -110,66 +187,82 @@ const UnconnectedVolunteers: React.FC<PropsFromRedux> = ({
             handleClick={(e) => handleVolunteerClick(e, volunteer)}
             volunteer={volunteer}
             key={volunteer.name}
-            isActive={volunteer.id === selectedVolunteer.id}
+            isActive={volunteer.id === volunteers.selected_volunteer.id}
           />
         ))}
       </section>
 
-      {selectedVolunteer.letters && (
-        <section className="d-flex flex-column p-5 m-5 bg-white shadow-sm w-50">
+      {volunteers.loading_details && spinner}
+
+      {problemLoadingDetails && (
+        <Container id="problem-loading-info">
+          <Card body>
+            There was a problem loading this volunteer's information.{' '}
+            {problemLoadingDetails}
+          </Card>
+        </Container>
+      )}
+
+      {volunteers.selected_volunteer.details && (
+        <section
+          id={page_id}
+          className="d-flex flex-column p-5 m-5 bg-white shadow-sm w-50">
           <span className="p3">Letters</span>
           <div className="d-flex flex-row">
-            <div className="d-flex flex-column">
+            <div className="d-flex flex-column letter-category">
               <span className="black-400 p4">In transit</span>
-              {selectedVolunteer.letters.map((letter) => (
-                <LetterCard
-                  key={letter.id}
-                  letter={letter}
-                  handleClick={(e) => handleLetterClick(e, letter)}
-                />
-              ))}
+              {volunteers.selected_volunteer.details.letters
+                .filter((letter) => letter.lob_status === 'letter.in_transit')
+                .map((letter) => (
+                  <LetterCard
+                    key={letter.id}
+                    letter={letter}
+                    handleClick={(e) => handleLetterClick(e, letter)}
+                  />
+                ))}
             </div>
-
-            <div className="d-flex flex-column ml-5">
+            <div className="d-flex flex-column ml-5 letter-category">
               <span className="black-400 p4">Delivered</span>
-              {selectedVolunteer.letters.map((letter, index) => (
-                <LetterCard
-                  letter={letter}
-                  key={letter.id}
-                  handleClick={(e) => handleLetterClick(e, letter)}
-                />
-              ))}
+              {volunteers.selected_volunteer.details.letters
+                .filter((letter) => {
+                  const now = new Date();
+                  return (
+                    letter.last_lob_status_update &&
+                    differenceInBusinessDays(
+                      now,
+                      letter.last_lob_status_update,
+                    ) >= 3 &&
+                    letter.lob_status === 'letter.processed_for_delivery'
```

Comment:
I think Jesse updated this snipped right so we can probably start using 'delivered'. 
We can change It later